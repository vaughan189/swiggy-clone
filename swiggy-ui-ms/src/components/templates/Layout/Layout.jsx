/**
 *
 * Layout
 *
 */
import React from 'react';
import PropTypes from 'prop-types';
import Header from '../../organisms/Header';


const Layout = ({ children }) => {

  return (
    <div>
      <Header />
      {children}
    </div>
  );
};

Layout.propTypes = {
  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
};

export default Layout;
