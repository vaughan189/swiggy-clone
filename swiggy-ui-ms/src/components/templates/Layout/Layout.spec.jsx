import React from 'react';
import { render } from '@testing-library/react';

import Layout from '.';

describe('Layout', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <Layout>
        <p>This is dummy page</p>
      </Layout>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
