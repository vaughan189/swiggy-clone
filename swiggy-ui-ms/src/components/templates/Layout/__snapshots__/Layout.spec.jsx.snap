// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular"
        >
          <h6
            class="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
          >
            Hiring & Staffing
          </h6>
        </div>
      </div>
    </header>
    <p>
      This is dummy page
    </p>
  </div>
</DocumentFragment>
`;
